src/layer_4/udp.rs

```diff
@@ -34,7 +34,7 @@ pub fn repl<'a, 'b>(
     client_info.port.dst = Some(udp_req.get_destination());
     let payload = udp_req.payload();
     let mut udp_repl;
-    if let Some(repl) = proto::repl(&payload, masscanned, &mut client_info) {
+    if let Some(repl) = proto::repl(&payload, masscanned, &mut client_info, None) {
```

Comment:
I am wondering if the tcb could not be stored in client_info. Thus, we wouldn't need a new parameter to `proto::repl`. Do you have any opinion?